src/Core/Content/LandingPage/Event/LandingPageIndexerEvent.php

```diff
@@ -10,19 +10,16 @@
 class LandingPageIndexerEvent extends NestedEvent
 {
     /**
-     * @var array
-     *
-     * @deprecated tag:v6.7.0 - Will be natively typed
+     * @var string[]
```

Comment:
please use `list<string>`, as it is more precise. `string[]` can also be `array<string, string>` or `array<int, string>`